lib/composer.json

```diff
@@ -5,6 +5,10 @@
         {
             "name": "Fatima Tariq",
             "email": "fatima@51degrees.com"
+        },
```

Comment:
Fatima should also be a part of engineering@51degrees.com, so no need to mention her explicitly :)

Reply:
Updated